ephios/core/templates/core/event_list.html

```diff
@@ -72,38 +88,197 @@ <h1>{% translate "Events" %}
+                        {% for shift in column_contents %}
+                            {% setvar request|participation_from_request:shift as participation %}
+                            {% setvar shift.signup_method.get_signup_stats as stats %}
+                            <div
+                                class="card day-calendar-shift-{{ shift.pk }} border-4
+                                       {% if stats.missing %}
+                                           border-danger-subtle
+                                       {% elif stats.free %}
+                                           border-warning-subtle
+                                       {% else %} border-success-subtle
+                                       {% endif %}">
+                                {% include "core/fragments/shift_header.html" with shift=shift participation=participation hide_date=True %}
+                                <div class="card-body">
+                                    {% include shift.signup_method with own_participation=participation %}
+                                </div>
+                                {# TODO stolen from event_box_big, dry up before merging #}
+                                <div class="card-footer">
+                                    {% if event.active %}
+                                        {% with can_sign_up=request|can_sign_up:shift can_decline=request|can_decline:shift can_customize_signup=request|can_customize_signup:shift %}
+                                            <form method="POST" class="form" id="signup-form-{{ shift.pk }}"
+                                                  action="{{ request|reverse_signup_action:shift }}">
+
+                                                {# If signup form is invalid, provide the shift times as initial values by including them in post data. #}
+                                                <input type="hidden" name="individual_start_time_0"
+                                                       value="{{ shift.start_time|date:"Y-m-d" }}">
+                                                <input type="hidden" name="individual_start_time_1"
+                                                       value="{{ shift.start_time|time:"H:i" }}">
+                                                <input type="hidden" name="individual_end_time_0"
+                                                       value="{{ shift.end_time|date:"Y-m-d" }}">
+                                                <input type="hidden" name="individual_end_time_1"
+                                                       value="{{ shift.end_time|time:"H:i" }}">
+
+                                                {% csrf_token %}
+                                                {% if can_sign_up %}
+                                                    <button type="submit" name="signup_choice" value="sign_up"
```

Comment:
Please add tests for rendering the day calendar and especially using the form submit here. (e.g. in case we change the required fields for the signup form in the future...)

Reply:
I'm not sure what You mean here. This is not new code, but instead stolen from from `event_big_box` and then you changed the form in `https://github.com/BenBals/ephios/pull/1`. So I'm not actually sure what the business logic is supposed to precisely achieve here. Therefore I'm not really comfortable asserting stuff about it's behaviour.

Reply:
This is copied from that other template, yes. There should be a test that submits this form here and assertes the participation is created with the correct state, nothing special. I can also write the tests if you don't feel like getting into our test infrastructure.

Reply:
That would be great, especially if this sounds straightforward to you.